client_test.go

```diff
@@ -124,3 +124,71 @@ func TestOneLogLine(t *testing.T) {
 	`
 	assert.Equal(t, `a\n b    c\n d\n\n efgh\n i\r\n\n j\t\n {"json":  "works",\n "f o o": "ba    r"\n }\n `, string(oneLogLine([]byte(in))))
 }
+
+func TestAPIRequestDCOS(t *testing.T) {
+	cases := []struct {
+		DCOSToken       string
+		ServerDCOSToken string
+		Username        string
+		Password        string
+		ServerUsername  string
+		ServerPassword  string
```

Comment:
We don't seem to use the username and password fields in any of the tests. I think we should just drop them from the cases struct and, subsequently, not include them into the configuration struct (explicitly).